From 44686ac3e7b536e905c3749814e4eb0e7e210440 Mon Sep 17 00:00:00 2001
From: Johannes Berg <johannes.berg@intel.com>
Date: Fri, 1 Sep 2023 07:50:02 +0200
Subject: [PATCH 23/28] iw: allow extra cflags

We can override the entirety of CFLAGS from the make
command line, but not add e.g. -Werror. Append a new
EXTRA_CFLAGS to make that possible.

Signed-off-by: Johannes Berg <johannes.berg@intel.com>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 2fb8db8..17be33f 100644
--- a/Makefile
+++ b/Makefile
@@ -18,6 +18,7 @@ CFLAGS += -Wall -Wextra -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-a
 CFLAGS += -Werror-implicit-function-declaration -Wsign-compare -Wno-unused-parameter
 CFLAGS += -Wdeclaration-after-statement
 CFLAGS += $(CFLAGS_EVAL)
+CFLAGS += $(EXTRA_CFLAGS)
 
 _OBJS := $(sort $(patsubst %.c,%.o,$(wildcard *.c)))
 VERSION_OBJS := $(filter-out version.o, $(_OBJS))
-- 
2.39.2

